### PR TITLE
IG-19957 added WA to coredns bug on upstream

### PR DESCRIPTION
**Pull request by yuribros1974**
IG-19957 adding master role to kubernetes

### CODE REVIEW

**Comment by yuribros1974** [comment]
Creating package of kubespray that include that change: https://jenkins-prod.iguazeng.com/job/kubespray/job/IG-19957_update_igz_offline/2/console
After that will create dedicated build that include that package, after that will build env with that

**Review by alexpirogovski — CHANGES_REQUESTED** [review]
roles/download/templates/kubeadm-images.yaml.j2
I would add a HUGE comment here so I don't forget to carry it to the next upstream update

File: roles/download/templates/kubeadm-images.yaml.j2 [hunk 1]
```diff
@@ -21,5 +21,5 @@ etcd:
 {% endif %}
 dns:
   type: CoreDNS
-  imageRepository: {{ coredns_image_repo | regex_replace('/coredns.*$','') }}
-  imageTag: {{ coredns_image_tag }}
+  imageRepository: k8s.gcr.io/coredns/coredns
+  imageTag: v1.8.0
```
Comment (alexpirogovski) [inline roles/download/templates/kubeadm-images.yaml.j2]: Should be taken from
https://github.com/iguazio/kubespray/blob/bbd0ade6f7ede7b3106097223fdb7bc35ff98c77/roles/igz_offline/defaults/main.yml#L12

**Comment by yuribros1974** [comment]
was tested on: https://jenkins-prod.iguazeng.com/job/customer_deploy/job/integ_3.2/347/
with kubespray: https://jenkins-prod.iguazeng.com/job/kubespray/job/IG-19957_update_igz_offline/6/
and build: https://jenkins-prod.iguazeng.com/job/build/job/development/1641/

**Review by uriblum46 — APPROVED** [review]

**Pull request merged**